packages/router/src/grpc/custody/authorize/assert-valid-swaps.ts

```diff
@@ -0,0 +1,19 @@
+import { Address } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
+import { TransactionPlan } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1/transaction_pb';
+import { Code, ConnectError } from '@connectrpc/connect';
+
+export const assertValidSwaps = (
```

Comment:
can we name this function to specifically indicate it checks that the user controls the claim address of the swap, since that's the purpose of it?

Reply:
Ah yeah, good call. I was thinking that we could eventually extend `assertValidSwaps` to do all kinds of assertions on swaps, but right now it's only doing one kind of assertion. [Fixed](https://github.com/penumbra-zone/web/pull/821/commits/f5a720d0ed3bd7ecca3a3ff85c3f0b8b44221f38).